velox/docs/functions/spark/map.rst

```diff
@@ -27,6 +27,13 @@ Map Functions
 
         SELECT map_from_arrays(array(1.0, 3.0), array('2', '4')); -- {1.0 -> 2, 3.0 -> 4}
 
+.. spark:function:: map_from_entries(struct(K,V)) -> map(K,V)
```

Comment:
Is the input argument `array(struct(K,V), struct(K,V), ...)`?

Reply:
updated.